django/db/backends/base/schema.py

```diff
@@ -433,6 +434,21 @@ def add_field(self, model, field):
         db_params = field.db_parameters(connection=self.connection)
         if db_params['check']:
             definition += " " + self.sql_check_constraint % db_params
+        if field.remote_field and self.connection.features.supports_foreign_keys and field.db_constraint:
+            constraint_suffix = "_fk_%(to_table)s_%(to_column)s"
+            # Add FK constraint inline, if possible
+            if self.sql_create_column_inline_fk:
+                to_table = field.remote_field.model._meta.db_table
+                to_column = field.remote_field.model._meta.get_field(field.remote_field.field_name).column
+                definition += " " + self.sql_create_column_inline_fk % {
+                    "name": self._fk_constraint_name(model, field, constraint_suffix),
+                    "column": self.quote_name(field.column),
+                    "to_table": self.quote_name(to_table),
+                    "to_column": self.quote_name(to_column),
+                }
+            # Otherwise, add FK constraints later
```

Comment:
Trailing dot is missing.

Reply:
See my previous comment. Would you like me to:

1. Add a trailing dot in the two places you indicated, so that _some_ of the comments in this function end in a period and others don't?
2. Update _all_ of the comments in this function, making changes to lines of code (and arguably polluting the commit history) that aren't related to this pull request?
3. Something else?

Reply:
Thanks for your effort :+1:. It is a kindly request for the first option. Trailing dot should be at the end of a sentence but there is no point in doing unrelated refactoring of existing code.

Reply:
Fixed.